src/features/completions.zig

```diff
@@ -151,7 +151,8 @@ fn nodeToCompletion(
     const datas = tree.nodes.items(.data);
     const token_tags = tree.tokens.items(.tag);
 
-    var doc_strings = orig_docs orelse std.ArrayListUnmanaged([]const u8){};
+    var doc_strings = std.ArrayListUnmanaged([]const u8){};
+    try doc_strings.appendSlice(arena, orig_docs);
```

Comment:
you are duplicating the `orig_docs` on every call of `nodeToCompletion` which can be avoided.